crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -421,6 +422,47 @@ where
             self.system_meta.name.as_ref(),
         );
     }
+    fn default_labels(&self) -> Vec<Box<dyn SystemLabel>> {
+        vec![Box::new(self.func.as_system_label())]
+    }
+}
+
+/// A [`SystemLabel`] that was automatically generated for a system on the basis of its `TypeId`.
+pub struct SystemTypeIdLabel<T: 'static>(PhantomData<fn() -> T>);
```

Comment:
This couldn't be `derive(Default)` because of the type parameter bounds, but should it still be `Default`?

Reply:
Interesting. This would enable people to construct these labels without an instance to a system type. Maybe thats good. Is there a use case?

Reply:
> Interesting. This would enable people to construct these labels without an instance to a system type. Maybe thats good. Is there a use case?

This could be useful if you wanted to initialize an array of these things, especially if we also impl'd Copy. I'm not immediately sure why you'd want to do that though.

Reply:
Haha I'm impatient, so I'm just going to merge this now. I'm happy to add this if we decide its needed.